modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackManager.kt

```diff
@@ -1591,6 +1591,10 @@ open class PlaybackManager @Inject constructor(
                                 } else {
                                     Timber.d("Episode is not downloaded $this")
                                 }
+                            },
+                            onError = {
+                                Timber.e(it, "Error observing episode")
+                                Sentry.captureException(it)
```

Comment:
I thought we would still want to send these to Sentry so that if a bunch of errors start getting hit in the app we're aware. I don't think this should do any harm to our Sentry quota since before this PR these kinds of errors were crashign the app, which got sent to Sentry. Let me know what you think.